CMakeLists.txt

```diff
@@ -65,12 +66,17 @@ find_package(PkgConfig REQUIRED)
 
 # glib...
 set(GLIB_MINIMUM 2.36)
-set (SERVICE_DEPS libayatana-common>=0.9.3 gio-unix-2.0>=${GLIB_MINIMUM} glib-2.0>=${GLIB_MINIMUM} gudev-1.0)
+set (SERVICE_DEPS libayatana-common>=0.9.3 gio-unix-2.0>=${GLIB_MINIMUM} glib-2.0>=${GLIB_MINIMUM})
 
 if (ENABLE_TESTS)
     list (APPEND SERVICE_DEPS properties-cpp>=0.0.1)
 endif ()
```

Comment:
I think that the above part needs to be addressed, too, for another reason. If I look at the code, I see that core/signal.h and core/property.h gets included in:

```
[sunweaver@sunobo ayatana-indicator-display (main)]$ grep -rE "(property.h|signal.h)"
tests/utils/glib-fixture.h:      g_signal_handler_disconnect(o, handler_id);
src/adbd-client.h:#include <core/signal.h>
src/usb-snap.h:#include <core/signal.h>
src/usb-monitor.h:#include <core/signal.h>
src/greeter.h:#include <core/property.h>
src/rotation-lock.cpp:    g_signal_handlers_disconnect_by_data(m_settings, this);
src/exporter.h:#include <core/signal.h>
src/usb-monitor.cpp:        g_signal_handlers_disconnect_by_data(m_udev_client, this);
src/indicator.h:#include <core/property.h>
```

Reply:
Amended CMakeLists.txt and src/CMakeLists.txt